cdc/owner.go

```diff
@@ -34,6 +35,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var markProcessorDownTime = time.Minute
```

Comment:
It seems the interval that processor updates `resolvedTs` or `checkpointTs` is not always less than 1 minute. Besides shall we allow several times update failure of processor and do we need to increase this value?